src/compiler/commandLineParser.ts

```diff
@@ -376,6 +375,63 @@ namespace ts {
         }
     }
 
+    function parseMultiValueStringArray(s: string, existingValue: string[]) {
+        let value: string[] = existingValue || [];
+        let hasError: boolean;

```

Comment:
as a matter of preference, can it be `hasError = false`. I'm not a big fan of returning undefined as primitive if `s` is falsy (though it is totally legit)